pkg/pod/network.go

```diff
@@ -80,3 +83,37 @@ func StaticIPBondAnnotationWithInterface(
 
 	return append(annotation, bond[0])
 }
+
+// StaticIPMultiNetAnnotation defines static ip address network annotation for pod object.
+func StaticIPMultiNetAnnotation(name string, ipAddr []string) *multus.NetworkSelectionElement {
+	return &multus.NetworkSelectionElement{
+		Name:      name,
+		IPRequest: ipAddr,
+	}
+}
+
+// StaticIPMultiNetDualStackAnnotation defines static ip address network annotation for pod object.
+func StaticIPMultiNetDualStackAnnotation(sriovNets, ipAddr []string) []*multus.NetworkSelectionElement {
+	annotation := []*multus.NetworkSelectionElement{}
+
+	// Verify ipAddr has an even number of IP addresses.
+	if len(ipAddr)%2 != 0 {
+		glog.V(100).Infof("ipAddr needs to contain an even number of IP addresses")
+		fmt.Println("ipAddr had an odd number of IP addresses")
+
+		return nil
```

Comment:
why nil, not error?

Reply:
I am only returning []*multus.NetworkSelectionElement - should I error?

Reply:
removed